lib/Transformer.js

```diff
@@ -2,22 +2,36 @@
  * Copyright (c) 2021 Digital Bazaar, Inc. All rights reserved.
  */
 import {CborldError} from './CborldError.js';
-import {KEYWORDS} from './keywords.js';
+import {KEYWORDS_TABLE} from './tables.js';
 
 export class Transformer {
   /**
    * Creates a new Transformer for transforming CBOR-LD <=> JSON-LD.
    *
-   * @param {object} options - The options to use when encoding to CBOR-LD.
+   * @param {object} options - The options to use.
    * @param {documentLoaderFunction} options.documentLoader -The document
    *   loader to use when resolving JSON-LD Context URLs.
-   * @param {Map} [options.appContextMap] - A map of JSON-LD Context URLs and
-   *   their encoded CBOR-LD values (must be values greater than 32767
-   *   (0x7FFF)).
+   * @param {Map} options.keywordsTable -A map of JSON-LD keywords
+   *  and their associated CBOR-LD integer values.
+   * @param {Map} options.stringTable - A map of string values and
+   *  their associated CBOR-LD integer values.
+   * @param {Map} options.urlSchemeTable - A map of URL schemes to
+   *  their associated CBOR-LD integer values.
+   * @param {Map} options.typedLiteralTable - A map of JSON-LD types
+   *  to maps that map values of that type to their associated CBOR-LD
+   *  integer values.
    */
-  constructor({appContextMap, documentLoader} = {}) {
-    this.appContextMap = appContextMap;
+  constructor({
+    documentLoader,
+    keywordsTable,
+    stringTable,
+    urlSchemeTable,
+    typedLiteralTable} = {}) {
     this.documentLoader = documentLoader;
+    this.termToId = keywordsTable;
```

Comment:
If we don't do this somewhere else -- we'll want to make sure we make a copy so that the caller's table isn't modified internally.

```suggestion
    this.termToId = new Map(keywordsTable);
```